__tests__/integration/one-app.spec.js

```diff
@@ -1381,9 +1381,6 @@ describe('Tests that require Docker setup', () => {
         date: [
           expect.any(String),
         ],
-        'expect-ct': [
-          'max-age=0',
-        ],
```

Comment:
https://owasp.org/www-project-secure-headers/#expect-ct

> Expect-CT
> Deprecated.
>
> ⚠️ Warning: This header will likely become obsolete in June 2021. Since May 2018 new certificates are expected to support SCTs by default. Certificates before March 2018 were allowed to have a lifetime of 39 months, those will all be expired in June 2021.